Auth0/JWTAlgorithm.swift

```diff
@@ -1,9 +1,5 @@
-#if WEB_AUTH_PLATFORM
```

Comment:
Why remove `WEB_AUTH_PLATFORM`? This is only used in macOS and iOS.

Reply:
huh, good point, I think I thought it was obj c related and so axed it - I'll put it back.

Also, should I add the same condition around `ChallengeGenerator.swift`?

Reply:
Yes, it should have the same condition. It prevents it being compiled on watchOS and tvOS.